### PR TITLE
[libgpg-error] Update to 1.42.

### PR DESCRIPTION
**Pull request by dcaliste**
An update is required to get a more recent GnuPG version.

Hopefully it's not breaking existing packages, thanks to @Thaodan hard work on migrating the pkgconfig label to follow the upstream naming scheme.

### CODE REVIEW

File: rpm/libgpg-error.spec [hunk 1]
```diff
@@ -1,10 +1,9 @@
 Summary: Library for error values used by GnuPG components
 Name: libgpg-error
-Version: 1.27
+Version: 1.42
 Release: 1
 URL: ftp://ftp.gnupg.org/gcrypt/libgpg-error/
 Source0: ftp://ftp.gnupg.org/gcrypt/libgpg-error/%{name}-%{version}.tar.bz2
-Patch0: libgpg-error-1.10-adding-pc.patch
 Group: System/Libraries
```
Comment (okodron) [inline rpm/libgpg-error.spec]: Please, drop the `Group:` as well.

File: rpm/libgpg-error.spec [hunk 2]
```diff
@@ -41,7 +40,6 @@ Man and info pages for %{name}.
 
 %prep
 %setup -q -n %{name}-%{version}/%{name}
```
Comment (okodron) [inline rpm/libgpg-error.spec]: Better:
```suggestion
%autosetup -n %{name}-%{version}/%{name}
```

File: rpm/libgpg-error.spec [hunk 3]
```diff
@@ -1,10 +1,9 @@
 Summary: Library for error values used by GnuPG components
 Name: libgpg-error
-Version: 1.27
+Version: 1.42
 Release: 1
 URL: ftp://ftp.gnupg.org/gcrypt/libgpg-error/
```
Comment (okodron) [inline rpm/libgpg-error.spec]: Can you change it to the actual one?
```suggestion
URL: https://www.gnupg.org/related_software/libgpg-error/
```

File: rpm/libgpg-error.spec [hunk 4]
```diff
@@ -1,10 +1,9 @@
...
 Source0: ftp://ftp.gnupg.org/gcrypt/libgpg-error/%{name}-%{version}.tar.bz2
-Patch0: libgpg-error-1.10-adding-pc.patch
 Group: System/Libraries
 License: LGPLv2+ and GPLv2+
```
Comment (okodron) [inline rpm/libgpg-error.spec]: Or just `LGPLv2+` for now?

File: rpm/libgpg-error.spec (hunk 1 above)
Comment (dcaliste) [reply rpm/libgpg-error.spec]: Good point, thanks. I've removed the Group: label in the other places.

File: rpm/libgpg-error.spec (hunk 2 above)
Comment (dcaliste) [reply rpm/libgpg-error.spec]: Replaced.

File: rpm/libgpg-error.spec (hunk 3 above)
Comment (dcaliste) [reply rpm/libgpg-error.spec]: Updated also.

File: rpm/libgpg-error.spec (hunk 4 above)
Comment (dcaliste) [reply rpm/libgpg-error.spec]: Good question, if I trust `grep -r SPDX-License-Identifier libgpg-error/`, then it's LGPLv2+ indeed (plus some another one for .in files). So I've removed the GPLv2+ tag.

**Review by okodron — APPROVED** [review]

**Comment by dcaliste** [comment]
Gentle ping, is it good to go in ? Does it need a bit more love before ?

**Review by pvuorela — APPROVED** [review]
Seems fine for me too. Sorry again for taking some time to check.

**Pull request merged**